### DISCUSSION TITLE
Please how to I use build process for static files for django deployment

### DISCUSSION THREAD

**Discussion by oyedejioyewole** [vercel/vercel#5187] [Help] [answered]
I have a django application I managed to deployed on your servers but I have installed whitenoise and some other stuff for static files hosting.

I then configured my `vercel.json` file for the deployment changes

```json
{
  "version": 2,
  "name": "application-name",
  "builds": [
    {
      "src": "project/wsgi.py",
      "use": "@ardnt/now-python-wsgi",
      "config": {
        "maxLambdaSize": "30mb"
      }
    },
    {
      "src": "build_files.sh",
      "use": "@vercel/static",
      "config": {
        "distDir": "staticfiles",
      }
    }
  ],
  "routes": [
    {
      "src": "/(.*)",
      "dest": "project/wsgi.py"
    }
  ]
}
```

`build_files.sh`

```sh
# build_files.sh
python manage.py collectstatic --noinput
```

So everything looks normal to me but when deploying I get this error that doesn't seem to go away even after **deleting** the file. Please how can I solve this the logs

```
Error! Couldn't parse JSON file current-folder/src/vercel.json.
```

**So how can I solve this is there like a repository you can suggest or something**

I use `Vercel` for deployments after running away from `Heroku`

_`I will be glad to provide more information`_

**Comment by bminde** [comment, marked as answer]
I think the problem is that it's not valid json. Try removing the comma after `"distDir": "staticfiles",`

**Reply by realitystevens** [reply]
What comma are you removing??
